src/app/components/code_block/code_block.html

```diff
@@ -14,8 +14,9 @@ <h6>Code</h6>
             </li>
         </ul>
         <div style="margin-top: 1px">
+            <!--- TODO: dynamically update "language-sql" or "language-python" based on model scope --->
             <pre style="background-color: white"
-                 class='code line-numbers'><code class="source-code highlight language-sql" ng-bind-html="highlighted"></code></pre>
+                 class='code line-numbers'><code class="source-code highlight language-python" ng-bind-html="highlighted"></code></pre>
```

Comment:
@drewbanin Any chance I could ask for your help here? :)

Big idea:
- If `mod.language == "python"`, we want this to be `language-python`
- If `mod.language == "sql"` (or undefined), we want this to be `language-sql`

I'm not enough of an Angular pro to know the ins and outs of `ng-class` & friends

Reply:
I think you can do
```
<code
    class="source-code highlight" ng-class="mod.language == 'python' ? 'language-python' : 'language-sql'"
    ng-bind-html="highlighted">
</code>
```

or, you could create [a scope variable](https://github.com/dbt-labs/dbt-docs/blob/main/src/app/components/code_block/code_block.js#L19-L20) in the component called `language_class` (or w/e) with a value of `language-sql` / `language-python` and then write:
```
<code class="source-code highlight" ng-class="language_class" ng-bind-html="highlighted">
</code>
```

Reply:
Thank you! I tried things sorta in this vein (random guesses), but I think the issue is that `mod`/`mod.language` isn't available in the `codeBlock` directive / in scope for `code_block.html`

<img width="1049" alt="Screenshot 2022-07-27 at 15 01 14" src="https://user-images.githubusercontent.com/13897643/181253224-04a7afc9-1fbc-4550-85ce-6b9025c94d61.png">